Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -265,6 +288,114 @@ - (void)urlSessionTask:(NSURLSessionTask *)sessionTask setState:(NSURLSessionTas
     SENTRY_LOG_DEBUG(@"SentryNetworkTracker finished HTTP span for sessionTask");
 }
 
+- (void)captureEvent:(NSURLSessionTask *)sessionTask
+{
+    NSHTTPURLResponse *myResponse = (NSHTTPURLResponse *)sessionTask.response;
```

Comment:
Some places do a type check and some other places don't, I am unsure if I should do it or not, I believe its not necessary otherwise it'd be a problem already.

Reply:
@brustolin should know. Please reflect this info as comments in the code, whatever the answer is.

Reply:
I did a type check just for safety and added a comment, other parts of the code don't btw, but I won't change that.